test/extended/ldap.go

```diff
@@ -0,0 +1,75 @@
+// +build ldap
```

Comment:
this will be test/extended/authentication/ldap.go after https://github.com/openshift/origin/pull/4256 gets merged